### ISSUE TITLE
hubot fails to start with phabrictor module

### ISSUE THREAD

**Issue by lloydoliver** [kemayo/hubot-phabricator#1]
Installed the hubot-phabricator module, set env vars for phabricator url, user, and arc cert.

Hubot fails to load with

```
[Fri Jun 26 2015 10:24:21 GMT+0000 (UTC)] ERROR Unable to load /data/hubot/node_modules/hubot-phabricator/src/phabricator: TypeError: Cannot call method 'replace' of undefined
  at module.exports (/data/hubot/node_modules/hubot-phabricator/src/phabricator.coffee:31:48, <js>:15:51)
  at Robot.loadFile (/data/hubot/node_modules/hubot/src/robot.coffee:253:11, <js>:177:13)
  at /data/hubot/node_modules/hubot-phabricator/index.coffee:12:11, <js>:26:33
  at Object.cb [as oncomplete] (fs.js:168:19)
```

Removing the phabricator module allows Hubot to start once again.

hubot@2.13.2 
hubot-phabricator@1.0.3

**Comment by kemayo** [comment]
Ah, I assumed that an ignore pattern would be set. One second and I can fix that.

**Comment by kemayo** [comment]
Okay, as of 1c9f835e2da68781f058fb51e3445949b03b9137 this is fixed. I've published 1.0.4 to npm with the fix as well.
